web_src/js/webcomponents/GiteaLocaleDate.js

```diff
@@ -0,0 +1,40 @@
+window.customElements.define('gitea-locale-date', class extends HTMLElement {
```

Comment:
why not extend `<relative-time>`?

Reply:
We aren't replacing https://github.com/github/relative-time-element, we are just adding another custom element to render absolute dates in locale format.

Reply:
Oh, and to answer specifically: `<relative-time>` is not meant to be passed dates, only datetimes. So it's currently unsuitable for this purpose and I'm not sure whether upstream would accept a `date` parameter as it would increase internal complexity of the component a lot.